CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master (unreleased)
 
+* Handle an upstream Ruby issue where the DidYouMean module is not available, which would break the `Rails/UnknownEnv` cop. ([@taylorthurlow][])
```

Comment:
Can you add `### Bug fixes` header and the PR's URL?

```suggestion
### Bug fixes

* [#297](https://github.com/rubocop-hq/rubocop-rails/pull/297): Handle an upstream Ruby issue where the DidYouMean module is not available, which would break the `Rails/UnknownEnv` cop. ([@taylorthurlow][])
```